Neos.Flow/Classes/Validation/Validator/MediaTypeValidator.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\Validation\Validator;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\ResourceManagement\ResourceMetaDataInterface;
+use Neos\Utility\MediaTypes;
+use Psr\Http\Message\UploadedFileInterface;
+
+/**
+ * The given $value is matches the defined medis types
```

Comment:
media types...

Reply:
😂 little late xD

how on earth did you find it now???

Reply:
looking though changes for 8.3 release I suppose?

Reply:
Indeed checked changes for the blog post ;)

The comment was more a reminder to adjust that ;)